all-of-us/workbench
# Chavanr/e2e cohort review participant detail

Expanded the cohort-review test case _to review the cohort data at participant level:_ 
added the following test steps to the test case: 
1. click the pen icon in the right grey panel
2. Choose a Review Status for Participant (exclude / include)
3. Go to the next participant 
4. Create a Review-Wide Annotation Field. (click on the annotations plus-icon)
5. Check review status on cohort review data table

Still pending:
6. Edit a Review-Wide Annotation Fields.
7. Delete a Review-Wide Annotation Field.

---
**PR checklist**

- [ ] I have run the E2E tests on ths change against my local UI + API server with `yarn test-local`
- [ ] If this includes a UI change, I have taken screen recordings or screenshots of the new behavior and notified the PO and UX designer
